mne/gui/_coreg_gui.py

```diff
@@ -159,6 +180,38 @@ class CoregModel(HasPrivateTraits):
     points_eval_str = Property(depends_on='point_distance')
 
     @cached_property
+    def _get_points(self):
+        with warnings.catch_warnings(record=True):  # comp to None in Traits
+            if self.points_filter is None:
+                self.points_filter = \
+                    np.ones(len(self.hsp.inst_points), np.bool8)
```

Comment:
The only `Property` that depends on `points_filter` is `points`, so `_get_points()` should not reassign `points_filter`, it should just read it.

Couldn't you just creat an index by
 - starting with `points_filter` (or if it's `None` uses `np.ones()`), 
 - and if EEG electrodes should be excluded use `np.logical_and`

or am I missing something?